lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -292,16 +292,11 @@ def test_org_query(self):
         for course in courses:
             self.enroll(course.id)
 
-        response = self.api_response(qargs={'org':'edX'})
+        response = self.api_response(data={'org':'edX'})
 
         # Verify only edX courses are returned
-        self.assertEqual(len(response.data), 3)
```

Comment:
You should keep this line, because otherwise the test will pass with an empty list for `response.data`.